### DISCUSSION TITLE
Submission Message as rich text

### DISCUSSION THREAD

**Discussion by Megafry** [verbb/formie#1681] [Ideas] [answered]
After successfully submitting a form, the submission message is displayed inside of a green `fui-alert`.
This doesn’t allow for much customization.

Would it be possible to create a new submit-action that replaces the form with text from a field with the following options:
- Bold
- Italic 
- Link

This is the desired output:
![image](https://github.com/verbb/formie/assets/10438940/67ceecb1-b2df-42c2-8bab-6db5cc273598)

**Comment by engram-design** [comment, marked as answer]
Styling the alert is completely up to you if you don't like the default Formie handling - best to use [Theme Config](https://verbb.io/craft-plugins/formie/docs/theming/theme-config#form-tags) to setup those classes.

Otherwise, the message can be rich text as defined in the form settings (you can add more [rich text buttons](https://verbb.io/craft-plugins/formie/docs/get-started/configuration#rich-text-configuration) if you need more formatting options) and you can set it to hide the form and just show the message.

**Reply by Megafry** [reply]
Thanks for your feedback
I approached the problem the wrong way, I will look into your suggestion.
Best regards.
